docs/source/PyMOLplugin.rst

```diff
@@ -0,0 +1,233 @@
+
+    **PyMOL Selection**: Enter the selection string which contains
+    waters (can contain other elements as well) in your PyMOL window. We
+    recommend selection contains system as well as waters of interest.
+
+    **Test Input**: Click this button to test the provided input.
+
+  .. tab:: **From Files**
+      
+    Provide trajectory and topology files. 
+
+    **Trajectory and Topology Files**: Browse and select the appropriate
+    files for your analysis. For list of supported file types check
+    MDAnalysis documentation `here
+    <https://www.mdanalysis.org/docs/documentation_pages/coordinates/init.html#supported-file-formats>`_.
+    File formats that can contain both trajectory and topology data such
+    as PDB, should only be provided using trajectory field.
+
+    **Every Nth Frame**: Specify how many frames to skip for analysis.
+
+    **Test Input**: Click this button to test the provided input.
+
+  .. tab:: **CWS Input**
+        
+    Use a pre-existing ConservedWaterSearch input. Provide the clustering data set and specify the number of frames.
+
+    **CWS input data file**: Browse and select the CWS input data file.
+    This data file should contain the clustering data set. If only
+    oxygens are clustered every row should contain the x, y, and z
+    coordinates of oxygen atoms for every frame to be studied. For
+    analysis of water types (recommended) each row should contain the
+    x, y, and z coordinates of oxygen and hydrogen atoms. For more
+    information see `ConservedWaterSearch docs
+    <https://conservedwatersearch.readthedocs.io/en/latest/examples.html>`_.
+    A CWS input data file can be generated when providing a PyMOL
+    selection or trajectory and topology files by using the **Export CWS
+    Input Data** button in the **Water Selection** section.
+
+    **Number of snapshots**: Number of snapshots in the trajectory used
+    to generate the CWS input data file.
+
+  .. tab:: **Restart**
+
+    Restart a previous water clustering analysis. Provide the partial
+    results file and CWS restart data file for restarting the analysis.
+
+    **Partial Results File**: Browse and select the partial results file
+    for restarting the analysis. This file should contain the partial
+    results of the clustering analysis. For more information see
+    `ConservedWaterSearch website 
+    <https://conservedwatersearch.readthedocs.io/en/latest/examples.html>`_.
+    
+    **CWS restart data file**: Browse and select the CWS restart data
+    file. This file has the same structure as CWS input data file.
+    This data file should contain the clustering data set. If only
+    oxygens are clustered every row should contain the x, y, and z
+    coordinates of oxygen atoms for every frame to be studied. For
+    analysis of water types (recommended) each row should contain the
+    x, y, and z coordinates of oxygen and hydrogen atoms. For more
+    information see `ConservedWaterSearch documentation
```

Comment:
In some instances we use ConservedWaterSearch docs and in some others we use documentation.